### PR TITLE
Client changes to accommodated server side master

### PR DESCRIPTION
**Pull request by hero101**
### Changes
- no longer saves from the client
- the server initializes the scene of the client on connect
- Whiteboard last saved subscription removed
- Last saved is received from the whiteboard server
- the client syncs the same way as before, receiving updates from the server (relayed from other clients)
- newly created clients have their scene initialized by the server

## Release Notes

- **New Features**
  - Introduced a new query for fetching whiteboard details without content.
  - Added hooks for executing the new query in React components.

- **Bug Fixes**
  - Removed outdated functionalities related to whiteboard content updates and subscriptions, streamlining the user experience.

- **Documentation**
  - Updated user interface feedback by removing references to saving actions, potentially simplifying user interactions.

- **Refactor**
  - Simplified the handling of whiteboard state and event notifications, focusing on remote save actions.

- **Chores**
  - Cleaned up unused code related to whiteboard content management and subscriptions.

### CODE REVIEW

File: src/domain/common/whiteboard/excalidraw/collab/Portal.ts [hunk 1]
```diff
@@ -70,12 +71,17 @@ class Portal {
...
-        transports: connectionOptions.polling ? ['websocket', 'polling'] : ['websocket'],
-        path: '/api/private/ws/socket.io',
-        retries: 0,
-        reconnection: false,
-      });
+      const socket = socketIOClient(
+        'localhost:4002',
+        // connectionOptions.url,
+        {
+          transports: connectionOptions.polling ? ['websocket', 'polling'] : ['websocket'],
```
Comment (hero101) [inline src/domain/common/whiteboard/excalidraw/collab/Portal.ts:78]: revert this later

File: src/domain/common/whiteboard/excalidraw/collab/Portal.ts [hunk 2]
```diff
@@ -70,12 +71,17 @@ class Portal {
...
       const { default: socketIOClient } = await import('socket.io-client');
 
-      const socket = socketIOClient(connectionOptions.url, {
-        transports: connectionOptions.polling ? ['websocket', 'polling'] : ['websocket'],
-        path: '/api/private/ws/socket.io',
-        retries: 0,
-        reconnection: false,
-      });
+      const socket = socketIOClient(
+        'localhost:4002',
```
Comment (hero101) [inline src/domain/common/whiteboard/excalidraw/collab/Portal.ts:75]: revert this later

File: src/domain/collaboration/whiteboard/containers/WhiteboardContentContainer.tsx
Comment (hero101) [inline src/domain/collaboration/whiteboard/containers/WhiteboardContentContainer.tsx:1]: do we still need this container?

File: src/domain/common/whiteboard/excalidraw/collab/Collab.ts [hunk 3]
```diff
@@ -175,15 +182,7 @@ class Collab {
...
-                  if (!this.portal.socketInitialized) {
-                    this.initializeRoom({ fetchScene: false });
-                    const remoteElements = decryptedData.payload.elements;
-                    const remoteFiles = decryptedData.payload.files;
-                    this.handleRemoteSceneUpdate(await this.reconcileElements(remoteElements, remoteFiles), {
-                      init: true,
-                    });
-                  }
-                  break;
+                  break; //todo remove
```
Comment (hero101) [inline src/domain/common/whiteboard/excalidraw/collab/Collab.ts:185]: is this needed?

**Review by ccanos — APPROVED** [review]

**Pull request merged**